design/api.md

```diff
@@ -549,6 +549,38 @@ TO BE IMPLEMENTED
 {% endapi-method-spec %}
 {% endapi-method %}
 
+{% api-method method="get" host="http://0.0.0.0" path="/api/v1/delete/:sess\_id" %}
```

Comment:
This section should be under the `admin` section, not the `web` section

Reply:
Sorry about that. I moved it.